banzai/images.py

```diff
@@ -166,6 +168,9 @@ def write_catalog(self, filename, nsources=None):
     def add_history(self, msg):
         self.header.add_history(msg)
 
+    def upate_datecrt(self):
```

Comment:
Why does this need its own method?

Reply:
I wanted to avoid directly modifying an object attribute.